editor/src/components/navigator/navigator-item/navigator-condensed-entry.tsx

```diff
@@ -288,6 +284,20 @@ const CondensedEntryItemContent = React.memo(
       ])
     }, [props.entry, dispatch, highlightedViews])
 
+    const getClickActions = useGetNavigatorClickActions(
```

Comment:
shouldn't this be using the clickable wrapper too?

Reply:
that's a negatory, because they would kick each other

Reply:
suboptimal, but if you look at the git history that's why I factored out the actions functions _and_ renamed the wrapper to `Navigator__Row__ClickableWrapper`